src/internal/Face.zig

```diff
@@ -217,12 +220,14 @@ fn capRound(
     self: Face,
     plotter_impl: *const PlotterVTable,
     clockwise: bool,
+    pen: ?Pen,
 ) PlotterVTable.Error!void {
     // We need to calculate our fan along the end as if we were
     // dealing with a 180 degree joint. So, treat it as if there
     // were two lines going in exactly opposite directions, i.e., flip the
     // incoming slope for the outgoing one.
-    var vit = self.pen.vertexIteratorFor(
+    debug.assert(pen != null);
```

Comment:
Not the biggest fan of this (or the other) assert but I didn't necessarily want to call this `InvalidState` either. Will be keeping an eye out for better options. However, this _did_ help in testing, where it caught the fact that we needed to lazy-allocate the pen during handling of `curve_to`, so it's doing its job. :P